scripts/auto-stop-idle/autostop.py

```diff
@@ -99,7 +99,12 @@ def get_notebook_name():
         else:
             idle = False
 else:
-    idle = False
+    client = boto3.client('sagemaker')
+    uptime = client.describe_notebook_instance(
+        NotebookInstanceName=get_notebook_name()
+    )['LastModifiedTime']
```

Comment:
```LastModifiedTime``` is when customer last updated the notebook instance itself, like stopping/starting notebook instance. It's not the last time when they used Jupyter.

Reply:
This should be simply change to ```True``` if nothing is returned back from /api/sessions, which means there's no active session.

Reply:
I disagree. 

If there are no active sessions, we should still wait the Idle time (from when the notebook is started) to shutdown the notebook. The only way to do that is to look at LastModifiedTime, as the value of when the notebook was started.